### PR TITLE
Update/yaml files sh byocs3

### PR DESCRIPTION
**Pull request by dthiex**
update SH BYOC yaml files with STAC JSON metadata

### CODE REVIEW

File: collections/alos-2-palsar-2-stripmap-for-economy.yaml [hunk 1]
```diff
@@ -19,11 +19,11 @@ Resolution: |
  SM1: 3m
 GeographicalCoverage: |
   Paris, France SM3: [49.004, 3.111 : 48.519, 3.672] (Upper Left: Bottom Right )  
-  Dunkirk, France SM3: [51.462, 1.775 : 54.447,3.186]   
-  LA SM1:  (34.226, -118.002 : 33.707, -117.869)  
-  Beijing, China SM1, SM3:  [40.129, 116.283 : 39.637, 117.247]  
-  Haneda SM1: (35.965, 139.651: 35.267, 139.897)
-TemporalAvailability: 2019-03-01  -  2020-12-31
+  Dunkirk, France SM3: [51.587, 1.127 : 50.466, 3.204]
```
Comment (Ardweaden) [inline collections/alos-2-palsar-2-stripmap-for-economy.yaml:22]: Where did this info come from?

File: collections/alos-2-palsar-2-stripmap-for-economy.yaml [hunk 2]
```diff
@@ -47,13 +47,75 @@ Tags:
...
     CollectionId: 5014c166-c13e-4bcf-b7cf-52d0312c54b8
-
+    Primary: true
+Extent:
+  spatial:
+    bbox:
+      -
+        - -118.002
+        - 33.707
+        - 117.247
```
Comment (Ardweaden) [inline collections/alos-2-palsar-2-stripmap-for-economy.yaml:65]: I think this should be `139.897`.

File: collections/e12c_motoway.yaml [hunk 3]
```diff
@@ -20,9 +20,46 @@ License: |
...
 Resources:
  - Group: Sentinel Hub Resources
-   EndPoint: shservices.mundiwebservices.com
+ - EndPoint: shservices.mundiwebservices.com
+   Name: Sentinel Hub
+   Role: processor
    Type: byoc-6d697640-a962-4eb0-8a59-30a8cb61dc0f
    CollectionId: 6d697640-a962-4eb0-8a59-30a8cb61dc0f
-
+   Primary: True
```
Comment (Ardweaden) [inline collections/e12c_motoway.yaml:28]: `True` should probably be lowercase. 

Also in other files, I won't write separate comments for all of them.

File: collections/e12c_motoway.yaml [hunk 4]
```diff
@@ -20,9 +20,46 @@ License: |
  [GNU General Public License v3.0](https://github.com/hfisser/Truck_Detection_Sentinel2_COVID19/blob/master/LICENSE)
 Resources:
```
Comment (Ardweaden) [inline collections/e12c_motoway.yaml:21]: Github doesn't allow me to comment on collapsed lines, but this collection has a mistake in the name I believe, (motoRway). Not connected to this PR, so fftr.

File: collections/jaxa_wq_chla_average.yaml [hunk 5]
```diff
@@ -48,9 +48,66 @@ License: |
...
+      - 45.8
+  t:
+    type: temporal
+    extent:
+      - '2018-04-07T00:00:00Z'
+      - null
+  band:
+    type: bands
+    values:
+      - chla_ave
```
Comment (Ardweaden) [inline collections/jaxa_wq_chla_average.yaml:111]: It seems to have 2 bands, `chla` and `chla_ave`.

File: collections/jaxa_wq_tsm_average.yaml [hunk 6]
```diff
@@ -48,9 +48,66 @@ License: |
...
+      - 45.8
+  t:
+    type: temporal
+    extent:
+      - '2018-04-07T00:00:00Z'
+      - null
+  band:
+    type: bands
+    values:
+      - tsm_ave
```
Comment (Ardweaden) [inline collections/jaxa_wq_tsm_average.yaml:111]: This one also has 2 bands, `tsm` and `tsm_ave`.

File: collections/population_density.yaml [hunk 7]
```diff
@@ -21,9 +21,46 @@ License: |
...
    CollectionId: a9743257-ef21-4fd3-999d-15c5c7fcacbd
-
+   Primary: True
+Extent:
+  spatial:
+    bbox:
+      -
+        - -180.0
+        - -90.0
+        - 179.99999999999983
```
Comment (Ardweaden) [inline collections/population_density.yaml:36]: Lol applicable https://xkcd.com/2170/

fftr

File: collections/tsmn_water_quality_saturday.yaml [hunk 8]
```diff
@@ -21,9 +21,46 @@ License: |
...
+      - 45.80052375793457
+  t:
+    type: temporal
+    extent:
+      - '2020-01-04T00:00:00Z'
+      - '2021-04-17T00:00:00Z'
+  band:
+    type: bands
+    values:
+      - tsmn
```
Comment (Ardweaden) [inline collections/tsmn_water_quality_saturday.yaml:64]: I believe the band name is `tsmnn`.

**Review by Ardweaden — CHANGES_REQUESTED** [review]

File: collections/alos-2-palsar-2-stripmap-for-economy.yaml (hunk 1 above)
Comment (dthiex) [reply collections/alos-2-palsar-2-stripmap-for-economy.yaml:22]: Dorothy checked the data. The corrected bbox is the one from the data. The original one in the file had a mistake and was not valid.

File: collections/population_density.yaml (hunk 7 above)
Comment (dthiex) [reply collections/population_density.yaml:36]: Haha, thought the same but as it's not our collection, I just copied the data that's already there...

File: collections/tsmn_water_quality_saturday.yaml (hunk 8 above)
Comment (dthiex) [reply collections/tsmn_water_quality_saturday.yaml]: Fixed

File: collections/jaxa_wq_tsm_average.yaml (hunk 6 above)
Comment (dthiex) [reply collections/jaxa_wq_tsm_average.yaml:112]: Added

File: collections/jaxa_wq_chla_average.yaml (hunk 5 above)
Comment (dthiex) [reply collections/jaxa_wq_chla_average.yaml:112]: Added

File: collections/e12c_motoway.yaml (hunk 4 above)
Comment (dthiex) [reply collections/e12c_motoway.yaml:21]: Spoke with Dorothy and it's a typo and was fixed here.

File: collections/e12c_motoway.yaml (hunk 3 above)
Comment (dthiex) [reply collections/e12c_motoway.yaml]: Changed in all files

File: collections/alos-2-palsar-2-stripmap-for-economy.yaml (hunk 2 above)
Comment (dthiex) [reply collections/alos-2-palsar-2-stripmap-for-economy.yaml]: True, changed

File: collections/e12c_motoway/README.MD [hunk 9]
```diff
@@ -1,4 +1,4 @@
-# E12C_Motoway
+# E12C_Motorway
```
Comment (Ardweaden) [inline collections/e12c_motoway/README.MD:1]: The folder name is still `e12c_motoway` btw. fftr

File: collections/e12c_motoway.yaml (hunk 4 above)
Comment (Ardweaden) [reply collections/e12c_motoway.yaml:21]: The folder name is still `e12c_motoway` btw. fftr

Comment (dthiex) [reply collections/e12c_motoway.yaml:21]: True, missed that. Changed, thanks.

**Pull request merged**